.circleci/notify_slack_channel

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+source $(dirname "$0")/define_build_environment_variables
+environment="$1"
+
+github_repo_url="${CIRCLE_REPOSITORY_URL//git@github.com:/https://github.com/}"
+github_repo_url="${github_repo_url//.git/}"
+
+cat <<END > payload.json
```

Comment:
_not a blocker_
🙋‍♂ Would be great to start moving these reused scripts to https://github.com/ministryofjustice/circleci-orbs or https://github.com/ministryofjustice/cloud-platform-tools-image/blob/master/circleci

Reply:
Ooh, shiny: https://github.com/CircleCI-Public/slack-orb

Reply:
These are great. We should definitely move towards these!